spec/controllers/tags_controller_spec.rb

```diff
@@ -464,45 +374,14 @@
         end
       end
 
-      context "when logged in as an admin with no role" do
-        let(:admin) { create(:admin) }
-
-        before { fake_login_admin(admin) }
-  
-        it "redirects with an error" do
-          put :update, params: { id: tag, tag: { syn_string: synonym.name }, commit: "Save changes" }
-          it_redirects_to_with_error(root_url, "Sorry, only an authorized admin can access the page you were trying to reach.")
-        end
+      subject { put :update, params: { id: tag, tag: { syn_string: synonym.name }, commit: "Save changes" } }
+      let(:success) do
+        it_redirects_to_with_notice(edit_tag_path(tag), "Tag was updated.")
+        tag.reload
+        expect(tag.merger_id).to eq(synonym.id)
       end
+      it_behaves_like "an action only fully authorized admins can access", authorized_roles: wrangling_full_access_roles
       
```

Comment:
Small formatting nitpick:
```suggestion

      it_behaves_like "an action only fully authorized admins can access", authorized_roles: wrangling_full_access_roles
```